### PR TITLE
Fixes attachment size issue

### PR DESCRIPTION
**Pull request by kidunot89**
…l layer.

### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------
- This moves the resolvers out of the models and into the `register_post_object_types` function. `srcSet` and `sizes` return varying values that rely on a provided parameter, and shouldn't be pre-queried in the mode.
- Cleans up of the `PostObject` model class, `register_post_object_types` function, and `get_post_object_fields` function and bring them into compliance with `WPCS` on everything except class name and filename.

Does this close any currently open issues?
------------------------------------------
#871 

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** Ubuntu 18.04

**WordPress Version:** 5.2

### CODE REVIEW

File: src/Type/Object/PostObject.php [hunk 1]
```diff
@@ -51,99 +72,131 @@ function register_post_object_types( $post_type_object ) {
...
+						'format' => [
+							'type'        => 'PostObjectFieldFormatEnum',
+							'description' => __( 'Format of the field output', 'wp-graphql' ),
+						],
 					],
+					'resolve'     => function( $source, $args ) {
+						if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+							// @codingStandardsIgnoreLine.
+							return $source->captionRaw;
+						} else {
```
Comment (kasn) [inline src/Type/Object/PostObject.php]: this unnecessary `else` block an be omitted

File: src/Type/Object/PostObject.php [hunk 2]
```diff
@@ -51,99 +72,131 @@ function register_post_object_types( $post_type_object ) {
...
+							'type'        => 'PostObjectFieldFormatEnum',
+							'description' => __( 'Format of the field output', 'wp-graphql' ),
+						],
 					],
+					'resolve'     => function( $source, $args ) {
+						if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+							// @codingStandardsIgnoreLine.
+							return $source->descriptionRaw;
+						} else {
+							// @codingStandardsIgnoreLine.
```
Comment (kasn) [inline src/Type/Object/PostObject.php]: this unnecessary else block an be omitted

File: src/Type/Object/PostObject.php [hunk 3]
```diff
@@ -212,13 +271,15 @@ function get_post_object_fields( $post_type_object ) {
 					'description' => __( 'Format of the field output', 'wp-graphql' ),
 				],
 			],
-			'resolve' => function( $source, $args ) {
+			'resolve'     => function( $source, $args ) {
 				if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+					// @codingStandardsIgnoreLine.
 					return $source->contentRaw;
 				} else {
```
Comment (kasn) [inline src/Type/Object/PostObject.php]: this unnecessary else block an be omitted

File: src/Type/Object/PostObject.php [hunk 4]
```diff
@@ -229,13 +290,15 @@ function get_post_object_fields( $post_type_object ) {
 					'description' => __( 'Format of the field output', 'wp-graphql' ),
 				],
 			],
-			'resolve' => function( $source, $args ) {
+			'resolve'     => function( $source, $args ) {
 				if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+					// @codingStandardsIgnoreLine.
 					return $source->titleRaw;
 				} else {
+					// @codingStandardsIgnoreLine.
```
Comment (kasn) [inline src/Type/Object/PostObject.php]: this unnecessary else block an be omitted

File: src/Type/Object/PostObject.php [hunk 5]
```diff
@@ -246,13 +309,15 @@ function get_post_object_fields( $post_type_object ) {
 					'description' => __( 'Format of the field output', 'wp-graphql' ),
 				],
 			],
-			'resolve' => function( $source, $args ) {
+			'resolve'     => function( $source, $args ) {
 				if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+					// @codingStandardsIgnoreLine.
 					return $source->excerptRaw;
 				} else {
```
Comment (kasn) [inline src/Type/Object/PostObject.php]: this unnecessary else block an be omitted

Comment (kidunot89) [reply src/Type/Object/PostObject.php]: fixed

File: src/Type/Object/PostObject.php (hunk 4 above)
Comment (kidunot89) [reply src/Type/Object/PostObject.php]: fixed

File: src/Type/Object/PostObject.php (hunk 3 above)
Comment (kidunot89) [reply src/Type/Object/PostObject.php]: fixed

File: src/Type/Object/PostObject.php (hunk 2 above)
Comment (kidunot89) [reply src/Type/Object/PostObject.php]: fixed

File: src/Type/Object/PostObject.php (hunk 1 above)
Comment (kidunot89) [reply src/Type/Object/PostObject.php]: fixed

**Comment by jasonbahl** [comment]
closes #871

**Pull request merged**